docs/releases/2.7.2.rst

```diff
@@ -20,22 +20,29 @@ Changes in Requirements
 Major Changes
 =============
 
-- Updated translations.
+- Bugfixes for some important issues.
+- Pulled latest translations.
 
 
 Below we provide much more detail. These are by no means exhaustive, view the
-`git log <https://github.com/translate/pootle/compare/stable/2.7.0...master>`_
+`git log <https://github.com/translate/pootle/compare/stable/2.7.1...2.7.2>`_
 for complete information.
 
 
 Details of changes
 ==================
 
+- Prevent local TM from crashing if elasticsearch is unavailable. elasticsearch
```

Comment:
It'd be good if `elasticsearch` was properly quoted with backticks — it's hard to read on the live release notes.